composer.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "spryker-eco/avalara-tax-shipment-tax",
+  "type": "library",
+  "description": "AvalaraTaxShipmentTax module",
+  "license": "MIT",
+  "require": {
+    "php": ">=7.3",
+    "spryker-eco/avalara-tax-extension": "^1.0.0",
```

Comment:
* Add `"spryker/transfer": "^3.25.0"` module to dependencies because of `*getOrFail()` usage.
* Copy the file from the module `Bundles/CategoryStorage/dependency.json`.

Reply:
Resolved.